apps/extension/src/App/Requester.tsx

```diff
@@ -0,0 +1,52 @@
+import browser from "webextension-polyfill";
+import { ExtensionKVStore } from "@anoma/storage";
+import {
+  ExtensionMessenger,
+  ExtensionRequester,
+  getAnomaRouterId,
+} from "extension";
+import { createContext, useContext, useEffect, useState } from "react";
```

Comment:
Not a big deal at all, but generally I try to keep third-party imports separate from local (e.g., `react`, etc.) - at least I've tried to do that and be relatively consistent. Do you have an opinion on that, or possibly think it isn't useful?

Reply:
So by separate you mean to group them and separate by empty line right?
Yeah I would do it too. I think I just got lazy with autoimports :D Will fix!